lib/game.rb

```diff
@@ -10,4 +14,34 @@ def initialize
     @player2 = nil
     @winner = nil
   end
+
+  def valid_move(index)
+    index.between?(1, 9) && @board.board_cell[index - 1].is_a?(Numeric)
+  end
+
+  def move(player)
+    puts "#{player.name}'s turn"
```

Comment:
- [ ] No `puts/print` and `gets.chomp` outside `main.rb` file. You should find and replace similar usage in your codebase.